### DISCUSSION TITLE
開發中的功能可以先不必deploy

### DISCUSSION THREAD

**Discussion by bestian** [3dw/auto20-next#154] [General]
我們有兩個網址可以用
https://auto20-next.pages.dev
放開發中的功能

https://we.alearn.org.tw

是公開網址，放已經成熟的功能

還在施工的功能，就只要build ，不要跑deploy.sh

以免一般用戶看到施工到一半的情況

這樣OK嗎？

**Comment by leechiuhui** [comment]
了解しました🫡
